detectron/utils/vis.py

```diff
@@ -96,7 +96,7 @@ def get_class_string(class_index, score, dataset):
 
 def vis_mask(img, mask, col, alpha=0.4, show_border=True, border_thick=1):
     """Visualizes a single binary mask."""
-
+    
```

Comment:
There are trailing spaces here again. Is your editor adding these automatically by any chance? You may want to configure your editor so that the trailing spaces are removed on save automatically (or at least highlighted as they are easy to miss otherwise).

Reply:
sorry I was editing the file in the browser, so it was easy to miss these trailing spaces. It should be removed now.